httpx/models.py

```diff
@@ -845,6 +846,19 @@ def cookies(self) -> "Cookies":
             self._cookies.extract_cookies(self)
         return self._cookies
 
+    @property
+    def links(
```

Comment:
Let's tighten up the type hints to not use `typing.Any` and add a docstring here.

Reply:
changed the typing to str and added a docstring